src/components/buttons/button-group/ButtonGroup.vue

```diff
@@ -8,6 +8,7 @@ type ModelType = ButtonProps['value'];
 export interface Props {
   vertical?: boolean;
   color?: ContextColorsType;
+  activeColor?: ContextColorsType;
```

Comment:
wonderful, this really makes sense instead of having to track it to the implementation.